lib/travis/build/script/csharp.rb

```diff
@@ -48,7 +61,7 @@ def export
         end
 
         def install
-          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution]
+          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution] && !is_mono_2?
         end
 
         def script
```

Comment:
unrelated to this PR, but just stumled over this: the xbuild line checks `if config[:solution]` again, this is redundant as we do it right above in the if.